requirements.txt

```diff
@@ -1,13 +1,26 @@
-# Last updated 2022-10-03
-apipkg>=3.0.1, <4.0.0
-chardet>=5.0.0, <6.0.0
-importlib_metadata>=5.0.0, <6.0.0
-more_itertools>=9.0.0, <10.0.0
-py>=1.11.0, <2.0.0
-pytest>=7.1.2, <8.0.0
-pytest-dependency>=0.5.1, <0.6.0 # versions below 1.0.0 are unstable, so we should only take in bugfixes automatically
-pytest-xdist>=3.0.2, <4.0.0
-pyyaml>=6.0.0, <7.0.0
-requests>=2.28.1, <3.0.0
-six>=1.16.0, <2.0.0
-zipp>=3.8.1, <4.0.0
+# Last updated 2022-10-31
```

Comment:
Does this change or affect our documented update procedure for requirements.txt ?

https://github.com/envoyproxy/nighthawk/blob/e610970a3ac4bbbb8e6cdacba2cd289af3110879/MAINTAINERS.md?plain=1#L94